data_processing/wdl/workflows/hifi_qc_workflow.wdl

```diff
@@ -2,50 +2,137 @@ version 1.0
 
 import "../tasks/read_stats.wdl" as read_stats_wf
 import "../tasks/ntsm.wdl" as ntsm_check
+import "../tasks/methylationCheck.wdl" as meth_check  
 
 workflow hifi_qc_wf {
     input {
-        Array[File] hifi_reads
+        File hifi_reads
         Array[File] other_reads
         String sample_id
+        Boolean perform_methylation_check = false  # Optional input to perform methylation check
```

Comment:
I think we should set the default to `true`.

I can imagine if we only run this when we expect methylation we could end up in a situation where we want to know if some sample's data has methylation and we don't know. I think it's nice to run this for all data and then just always know. 

That's just my quick-and-dirty take, though.